data/src/main/java/com/linkedin/util/DataComplexUtil.java

```diff
@@ -0,0 +1,187 @@
+package com.linkedin.util;
+
+import com.linkedin.data.Data;
+import com.linkedin.data.DataList;
+import com.linkedin.data.DataMap;
+import com.linkedin.data.DataMapBuilder;
+import com.linkedin.data.collections.CheckedUtil;
+import com.linkedin.data.template.DataTemplateUtil;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+
+/**
+ * Utils to convert generic List/Map to DataList/DataMap or vice versa
+ */
+public class DataComplexUtil {
```

Comment:
Can you move the relevant unit test from here?

https://git.corp.linkedin.com:1367/a/plugins/gitiles/multiproducts/graphql-infra/+/master/util/src/test/java/com/linkedin/graphql/DataComplexUtilTest.java